README.md

```diff
@@ -16,6 +16,23 @@
 
 ### Install
 
+**With NPM**
+* Install the dependency:
+
+   ```javascript
+   npm install --save ng-mask
+   ```
+* Add ngMask.min.js to your code:
+
+   ```html
+   <script src='node_modules/ng-mask/dist/ngMask.min.js'></script>
```

Comment:
really?

Reply:
Really. I don't even know how I found it on NPM back when I sent this PR.